scripts/activate_cached-filter.sh

```diff
@@ -0,0 +1,2 @@
+git config filter.cached-output.clean "bash scripts/cached_output_clean.sh"
```

Comment:
Let's make the scripts executable and use `./scripts/cached_output_clean.sh`. This would also require that you need to add an executable flag to the files, and add the shell specification to the first line.